server/app/views/admin/questions/NorthStarQuestionPreview.java

```diff
@@ -0,0 +1,125 @@
+package views.admin.questions;
+
+import auth.CiviFormProfile;
+import com.google.auto.value.AutoValue;
+import com.google.inject.Inject;
+import controllers.AssetsFinder;
+import controllers.LanguageUtils;
+import controllers.applicant.ApplicantRoutes;
+import java.util.Optional;
+import modules.ThymeleafModule;
+import org.thymeleaf.TemplateEngine;
+import play.i18n.Messages;
+import play.mvc.Http.Request;
+import services.DeploymentType;
+import services.applicant.ApplicantData;
+import services.applicant.ApplicantPersonalInfo;
+import services.applicant.question.AddressQuestion;
+import services.applicant.question.ApplicantQuestion;
+import services.program.ProgramQuestionDefinition;
+import services.question.exceptions.UnsupportedQuestionTypeException;
+import services.question.types.QuestionDefinition;
+import services.question.types.QuestionType;
+import services.settings.SettingsManifest;
+import views.applicant.NorthStarApplicantBaseView;
+import views.questiontypes.ApplicantQuestionRendererFactory;
+import views.questiontypes.ApplicantQuestionRendererParams;
+import views.questiontypes.ApplicantQuestionRendererParams.ErrorDisplayMode;
+
+public class NorthStarQuestionPreview extends NorthStarApplicantBaseView {
```

Comment:
since it is getting used on the admin side, I think NorthStarApplicantBaseView should be renamed NorthStarBaseView and moved to a non-applicant directory

Reply:
although I guess you're not actually including any admin side TS or anything, so I'm a little torn on that

Reply:
I support making it Applicant/Admin neutral. It can be confusing what things are applicant facing vs admin facing in our code so anything we can do to clarify that is good 👍

Reply:
Renamed to NorthStarBaseView